lib/trackler/problem.rb

```diff
@@ -7,81 +7,108 @@ class Problem
     def initialize(slug, root)
       @slug = slug
       @root = root
+      @xcommon_repository_url = 'https://github.com/exercism/x-common/blob/master'
     end
 
     def exists?
-      !!md && !!yaml
+      description_exists? && yaml_exists?
     end
 
     def name
-      @name ||= slug.split('-').map(&:capitalize).join(' ')
+      @name ||= slug_to_name
```

Comment:
Do we use this in more than one place? I don't see any reason to have an extra method for it.